### PR TITLE
Create Interoperability Precis.md

### PR DESCRIPTION
**Pull request by jcstine**

### CODE REVIEW

File: folderAWG/Interoperability Precis.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+V. Vocabulary: Working Definitions 
+● Conversational access point: the means of carrying a signal to a conversational platform. Users initiate and conduct conversations through access points. An access point can be physical or virtual. Current examples of conversational access points include smartphones, smart speakers, personal computers, wearables, or microphone-enabled smart home hubs. 
+○ In the interoperable world, conversational access points will enable connection to any conversational platform and agent. 
+● Conversational platform: the combination of components that enables the operation and management of one or more conversational agents. These components may include artificial intelligence-powered natural language understanding, natural language generation, and dialog management. Current examples of conversational platform providers are Amazon, Google, Xiaomi, Nuance, SoundHound, Rasa, Cerence, Mycroft, and many others4.
```
Comment (pmotch) [inline folderAWG/Interoperability Precis.md:29]: There's a typo at the end of the sentence. A `4` after "others".

File: folderAWG/Interoperability Precis.md [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+● Users are unable to communicate directly with and through other agents, or access capabilities developed on other platforms. 
+● Users—individual and organizational users alike— are expected to share all data with the platform provider, which could include conversational, tonal, biometric, and biomarker data. 
+● Users must surrender credentials to access in-platform capabilities. 
+● Sub-agent and capability creators and voice innovators are limited to the technology dictates of the proprietary platforms. 
+
+The lack of interoperability between market-leading, proprietary conversational agents requires that organizational users of voice must develop and maintain multiple, proprietary systems, or invest in agent translation and management software. This is among the many reasons a recent European Commission sector inquiry5found that proprietary conversational platforms and their agents limit the market potential and current growth of home-based Internet of Things (IOT) systems.
```
Comment (pmotch) [inline folderAWG/Interoperability Precis.md:45]: Typo: `inquiry5found`

Comment (pmotch) [reply folderAWG/Interoperability Precis.md:45]: Also, banal, but we say `IOT` above, then here we say `Internet of Things (IOT)`. The latter should probably come first, then "IOT" can be used as an abbreviation from there on out.

File: folderAWG/Interoperability Precis.md [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+● Users are unable to communicate directly with and through other agents, or access capabilities developed on other platforms. 
+● Users—individual and organizational users alike— are expected to share all data with the platform provider, which could include conversational, tonal, biometric, and biomarker data. 
+● Users must surrender credentials to access in-platform capabilities. 
+● Sub-agent and capability creators and voice innovators are limited to the technology dictates of the proprietary platforms. 
+
+The lack of interoperability between market-leading, proprietary conversational agents requires that organizational users of voice must develop and maintain multiple, proprietary systems, or invest in agent translation and management software. This is among the many reasons a recent European Commission sector inquiry5found that proprietary conversational platforms and their agents limit the market potential and current growth of home-based Internet of Things (IOT) systems. 
+The benefits of agent-to-agent interoperability:
```
Comment (pmotch) [inline folderAWG/Interoperability Precis.md:46]: Several sub-bullets appear to need a new line in many of the main bullets in this section.

File: folderAWG/Interoperability Precis.md [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+We believe the future of voice assistance is a voice-expanded World Wide Web, made of: 
+● general purpose, proprietary platform-based conversational agents 
+● brand-, title-, character- enterprise- and organization-specific conversational agents ● sensors, aggregators, and AI’s within smart, IOT environments 
+● capabilities with generic, reusable functions that can be used to accelerate the development of conversational agents. 
+
+### V. An Interoperable Voice Agent Should... 
+● Reuse conversational capabilities from other developers, including those running on other conversational platforms 
+● Share dialog, data, context, and controls with other voice agents, regardless of platform or technological parentage 
+● Be addressable directly through an explicit user request 
+● Make itself available to meet a user request on behalf of another trusted voice agent ● Support security and privacy by design.
```
Comment (pmotch) [inline folderAWG/Interoperability Precis.md:63]: Another new line needed.

File: folderAWG/Interoperability Precis.md [hunk 5]
```diff
@@ -0,0 +1,128 @@
...
+● Reuse conversational capabilities from other developers, including those running on other conversational platforms 
+● Share dialog, data, context, and controls with other voice agents, regardless of platform or technological parentage 
+● Be addressable directly through an explicit user request 
+● Make itself available to meet a user request on behalf of another trusted voice agent ● Support security and privacy by design. 
+
+### VI. The Open Voice Network Interoperability Initiative: From Talk to Action 
+To develop the standard message formats, protocols, and data formats necessary for agent-to-agent interoperability, the Open Voice Network will cut the issue into layers. Presently, we envision six layers, termed “enhancements,” that operate atop a standardized communication infrastructure. 
+This infrastructure will include: 
+● a set of well-defined messages for starting, stopping, suspending, and resuming voice agents; 
+● protocols consisting of sequences of messages that implement the enhancement; and, ● a set of self-defined data formats for transferring data among agents.
```
Comment (pmotch) [inline folderAWG/Interoperability Precis.md:69]: New line needed

File: folderAWG/Interoperability Precis.md [hunk 6]
```diff
@@ -0,0 +1,128 @@
...
+4. Processing implicit requests. We define “implicit” as a request that searches across voice agent services rather than seeking specific, named voice agent destination. This will rely upon an arbitration service to select (or recommend) a voice agent from many. 
+5. Sharing data and context between voice agents. This enhancement advances #3, above, and enables agents to manage the sharing of data, context, and controls. It also will enable agents to call upon the universe of conversational capabilities, regardless of platform parentage. 
+6. Generating and operating a private agent persona. A persona refers to the voice and characteristics presented by a conversational agent. Such agents have a distinctive sound and ambiance that leads to branding, the promotion of a particular enterprise by means of advertising, and distinctive design. This enhancement will enable an enterprise voice agent to fully express a specific brand tone, and enable capabilities to adapt to or adopt the persona of the enterprise agent. 
+Other possible enhancements include:
```
Comment (pmotch) [inline folderAWG/Interoperability Precis.md:78]: Without an extra return here, this line will appear in line with the previous paragraph.

File: folderAWG/Interoperability Precis.md [hunk 7]
```diff
@@ -0,0 +1,128 @@
...
+5. Sharing data and context between voice agents. This enhancement advances #3, above, and enables agents to manage the sharing of data, context, and controls. It also will enable agents to call upon the universe of conversational capabilities, regardless of platform parentage. 
+6. Generating and operating a private agent persona. A persona refers to the voice and characteristics presented by a conversational agent. Such agents have a distinctive sound and ambiance that leads to branding, the promotion of a particular enterprise by means of advertising, and distinctive design. This enhancement will enable an enterprise voice agent to fully express a specific brand tone, and enable capabilities to adapt to or adopt the persona of the enterprise agent. 
+Other possible enhancements include: 
+● discovery—locating an agent; 
+● planning—developing a sequence of voice agents and capabilities that perform a complex task; and, 
+● identification and authentication of both the user and desired conversational agent, to allow a secure connection. 
+These and other potential enhancements are currently outside of the scope of the Open Voice Network Voice Agent Interoperability initiative.
```
Comment (pmotch) [inline folderAWG/Interoperability Precis.md:82]: Probably want a new line before this sentence. Will appear in the previous bullet.

File: folderAWG/Interoperability Precis.md [hunk 8]
```diff
@@ -0,0 +1,128 @@
...
+● identification and authentication of both the user and desired conversational agent, to allow a secure connection. 
+These and other potential enhancements are currently outside of the scope of the Open Voice Network Voice Agent Interoperability initiative. 
+
+### VII. How You Can Accelerate the Open Voice Network Agent-to-Agent Initiative 
+To accelerate the Open Voice Network agent-to-agent interoperability initiative, please consider helping to shape the future as you: 
+● Invest your expertise in the Open Voice Network interoperability Initiative: ○ Participate in weekly debates, discussions, and reviews. Meetings are virtual, recorded, no more than 60 minutes, and, in keeping with OVON culture, they begin and end on time. 
+○ Review and contribute to the work as your schedule permits. Documents (including this one) can be found in the Open Voice Network GitHub repository. Should you see a reason for revision, simply open a pull request. And, you can keep up with the weekly debates and discussions by reviewing the meeting recordings. They’re stored in the Open Voice Network Google Workspace, accessible by members and by requesting permission.
```
Comment (pmotch) [inline folderAWG/Interoperability Precis.md:87]: A proper way to say it: "...can be found in the Open Voice Network GitHub organization in the docs repository"

File: folderAWG/Interoperability Precis.md [hunk 9]
```diff
@@ -0,0 +1,128 @@
...
+To accelerate the Open Voice Network agent-to-agent interoperability initiative, please consider helping to shape the future as you: 
+● Invest your expertise in the Open Voice Network interoperability Initiative: ○ Participate in weekly debates, discussions, and reviews. Meetings are virtual, recorded, no more than 60 minutes, and, in keeping with OVON culture, they begin and end on time. 
+○ Review and contribute to the work as your schedule permits. Documents (including this one) can be found in the Open Voice Network GitHub repository. Should you see a reason for revision, simply open a pull request. And, you can keep up with the weekly debates and discussions by reviewing the meeting recordings. They’re stored in the Open Voice Network Google Workspace, accessible by members and by requesting permission. 
+● Raise your voice on behalf of the Open Voice Network interoperability initiative: ○ Join the Open Voice Network Ambassador corps. OVON Ambassadors participate regularly in public forums, whether through social media, social audio, or industry events. You’ll have the support of the OVON’s Outreach Team.
```
Comment (pmotch) [inline folderAWG/Interoperability Precis.md:88]: Other sub-bullets needed newlines in this line and the line after.

**Review by pmotch — CHANGES_REQUESTED** [review]
Lots of markdown fixes needed. You can always click the "preview" tab to make sure it renders as you prefer. There are also a lot of great Markdown editors you can use to do live editing as well; something to consider.

**Comment by pmotch** [comment]
Also noticed its in a folder called `folderAWG`; unsure if that's on purpose or not. Also, did you mean to call the filename "Create Interoperability Precis.md". Just checking in case.

**Comment by JohnTrammellTarget** [comment]
@jcstine is the work in this PR superseded by the other précis work done elsewhere?

**Comment by JohnTrammellTarget** [comment]
Will be resubmitting as PDF

**Pull request closed without merging**